tests/Fixture/Tool/ToolWithToolParameterAttribute.php

```diff
@@ -21,45 +21,45 @@ final class ToolWithToolParameterAttribute
      * @param object $shippingAddress  The shipping address given to the tool
      */
     public function __invoke(
-        #[ToolParameter(
+        #[With(
```

Comment:
As this is now quite short, what about putting the entire attribute in one line?